### DISCUSSION TITLE
[MSG] yarprun successfully executed on icub-head,  but status is unavailable, icon didn't turn green

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#622] [Q&A] [answered]
https://icub-tech-iit.github.io/documentation/icub_starter_kits/first_steps/#laptop-commands
On nodes :
    please select both icubsrv and icub-head
    click GREEN play button. 
    Both TV icon should turn from RED to GREEN

i followed the guide steps, only icubsrv icon turn from red to green,  icub-head icon didn't change but [MSG] yarprun successfully executed on icub-head.
As in below pic

![2023-05-06 21-44-46 的屏幕截图](https://user-images.githubusercontent.com/7019692/236628585-471de413-8388-40ec-a54d-5665449b7721.png)

**Comment by jimingda** [comment]
I think i have fixed this problem by press the Motor Lighting Indicator on iCub before launch yarpserver and yarpmanager.

**Reply by pattacini** [reply, marked as answer]
The motor button turns on the embedded boards, so it has nothing to do with `yarprun`, which instead runs abord the `icub-head` machine. Thus, you ought to make sure that `icub-head` is up and running (the CPU button is 🟢) and that there are no other instances of `yarprun` ("zombie" processes that you can check via `ps aux`).

Also, `icub-head` needs to see the YARP server, so you can check that out by doing:
```console
yarp where
```
from a terminal on `icub-head`.
